build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -182,11 +186,19 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 		const branchName = `${flags.source}-${flags.target}-${shortCommit(prHeadCommit)}`;
 		this.branchesToCleanup.push(branchName);
 
-		this.verbose(
+		this.log(
 			`Creating and checking out branch: ${branchName} at commit ${shortCommit(
 				prHeadCommit,
 			)}`,
 		);
+
+		await this.gitRepo.gitClient.raw([
+			"config",
+			"--global",
+			`url.https://${flags.pat}@github.com/.insteadOf`,
+			"https://github.com/",
```

Comment:
This is not obvious, so I'd put the info from the PR description in a comment. Is there documentation on this behavior somewhere that we could link to as well?

Reply:
I don't have link to the documentation, I found a solution similar to your command through Stack Overflow. This can be set in the workflow as well.